### ISSUE TITLE
Update and delete operations

### ISSUE THREAD

**Issue by LunarMarathon** [LunarMarathon/graphToTable#28]
I'm assigning this to myself for now, if anyone would like to try it instead, let me know

**Comment by Savio629** [comment]
Can you explain it in brief?
You need a option to delete the selected node from the table and then update it?
I can try to add this feature!

**Comment by LunarMarathon** [comment]
Yep. But it seems kinda complex since there's so much to this 

1. Deleting the whole row - but when this is done, the key node (first node of the row to be deleted) needs to be deleted in other rows where it's there
2. Edit node - So again if one node is changed, ex. Zoos to Zeus (lol) then the changes should appear in other nodes too - like finding all nodes with Zoos and replacing with Zeus
3. Deleting node - wait, this isn't same as 1, say I added Node1-Oceans, Node2-Zeus and I just want to remove this connection (Zeus and Oceans both have their own rows with correct data which shouldn't be deleted) So here, from Zeus row, Oceans should be removed and from Oceans row, Zeus should be removed(should be as though they weren't added at all - reverting addition of these two nodes to the table - not to be mistaken with undo btn/functionality, should be able to del whenever needed

**Comment by LunarMarathon** [comment]
I hope I made sense ;-; let me know if you have any doubts. Go through the code once too and let me know. We can discuss the ui part for that accordingly

**Comment by Savio629** [comment]
Yeah I got the idea about what you are trying to explain. The thing is I have an idea on how nodes work and how such operations work with it. But I need to implement is js and also taking care of all of the other stuff going with it(searching, generating).
I'll go through the code properly and update you. If you get any idea on how to implement it just update me to...

**Comment by LunarMarathon** [comment]
Yeah, sure thing! I was checking CRUD examples implemented with JS but I also feel like a database would be better and easier for querying so I'm learning about that

**Comment by Savio629** [comment]
Yeah! So I would like to contribute on this project even after the hacktober-fest.
I'll also try some different approach on this and will update you with the same.

**Comment by LunarMarathon** [comment]
That's good to hear! but unfortunately I might be halting this project for some time. You can go through the last merged code once if you're interested, I added edit cell and delete row code. Del code does only half the stuff now - just del the row and not the nodes present in other rows as I mentioned in 1 above. 

I'll let you know again if I change my mind, tysm!

**Comment by Savio629** [comment]
Okay no problem!
